packages/pcd/pod-pcd/src/PODPCD.ts

```diff
@@ -0,0 +1,247 @@
+import {
+  DisplayOptions,
+  ObjectArgument,
+  PCD,
+  PCDPackage,
+  SerializedPCD,
+  StringArgument
+} from "@pcd/pcd-types";
+import { POD, PODEntries } from "@pcd/pod";
+import { requireDefinedParameter } from "@pcd/util";
+import JSONBig from "json-bigint";
+import { v4 as uuid } from "uuid";
+
+/**
+ * A set of entries defining a POD, represented in an object.  See `@pcd/pod`
+ * for full definition and documentation.
+ */
+export type { PODEntries } from "@pcd/pod";
+
+/**
+ * The globally unique type name of the {@link PODPCD}.
+ */
+export const PODPCDTypeName = "pod-pcd";
+
+/**
+ * Interface containing the arguments that 3rd parties use to
+ * initialize this PCD package.
+ * It is empty because this package does not implement the `init` function.
+ */
+export interface PODPCDInitArgs {}
```

Comment:
> Meanwhile I discovered elsewhere that this isn't really the way to represent an empty type (because of how structural typing works), so actually it's equivalent to any

I think in this case using the `never` type would work when parameterizing `PCDPackage`. Another type you should be aware of is `unknown`

Reply:
>  Also I notice there's some inconsistency in whether these types use type vs. interface. Is there a consistent pattern you'd suggest following here?

yeah, we've been inconsistent. I don't think there's any practical difference so I've just been doing whatever is locally consistent. Ideally the whole codebase is consistently using one vs the other, but I don't really care that much.

Reply:
`never` isn't accepted due to the existence of a placeholder call to `init` in passport-client, which I copied from what you did for MessagePCD.  `unknown` is accepted, so I'll go with that.